libs/rbcapi/src/bills/index.ts

```diff
@@ -0,0 +1,69 @@
+import { BillResult, ProgressCallback } from '@thecointech/bank-interface';
+import { Page } from 'puppeteer';
+import { ApiAction } from '../action';
+import { Decimal } from 'decimal.js-light';
+import { addPayee } from './managePayees';
+import { log } from '@thecointech/logging';
+
+//
+// Pay bill
+// If nickname exists, use this a payee, else create a new payee and set it's nickname as passed.
+export async function payBill(prefix: string, nickname: string, payee: string, accountNo: string, amount: Decimal, _progressCb?: ProgressCallback): Promise< BillResult > {
+
+  const act = await prepareBillPayee(prefix, nickname, payee, accountNo);
+
+  await act.page.type('#amount', amount.toString());
+  await act.clickAndNavigate('#id_btn_submit', 'Bill Submit');
+  await act.clickAndNavigate('#id_btn_confirm', 'Bill Confirm');
```

Comment:
these 3 lines are the only ones not tested, as I dont really wanna pay a bill every jest run

Reply:
You could auto pay all your bills that way 🤣

Reply:
It would then be doubly-bad if I miss my monthly due-dates; Visa is mad at me, and my testing is broken!